### PR TITLE
add form name caution to choice_name option

### PR DESCRIPTION
**Pull request by xabbuh**
see symfony/symfony#30398

### CODE REVIEW

File: reference/forms/types/options/choice_name.rst.inc [hunk 1]
```diff
@@ -9,3 +9,8 @@ of the choice views in the template.
 
 This can be a callable or a property path. See `choice_label`_ for similar usage.
 If ``null`` is used, an incrementing integer is used as the name.
+
+caution::
```
Comment (javiereguiluz) [inline reference/forms/types/options/choice_name.rst.inc]: This should be `.. caution::` :)

File: reference/forms/types/options/choice_name.rst.inc [hunk 2]
```diff
@@ -9,3 +9,8 @@ of the choice views in the template.
...
+
+    The configured value must be a valid form name. Make sure to only return
```
Comment (javiereguiluz) [inline reference/forms/types/options/choice_name.rst.inc]: Do we have a link to some article that explains what is a "valid form name" ?  Or can we explain what's that in a short sentence? Thanks.

Comment (xabbuh) [reply reference/forms/types/options/choice_name.rst.inc]: I have pushed an update. What do you think about the changed text?

Comment (javiereguiluz) [reply reference/forms/types/options/choice_name.rst.inc]: Now it's perfectly explained! Thanks.

**Comment by javiereguiluz** [comment]
Thank you Christian.

**Pull request merged**